core/src/test/java/org/elasticsearch/index/IndexingSlowLogTests.java

```diff
@@ -62,6 +63,13 @@ public void testSlowLogParsedDocumentPrinterSourceToLog() throws IOException {
         p = new SlowLogParsedDocumentPrinter(index, pd, 10, true, 3);
         assertThat(p.toString(), containsString("source[{\"f]"));
         assertThat(p.toString(), startsWith("[foo/123] took"));
+
+        // Throwing a error if source cannot be converted
+        source = new BytesArray("invalid");
+        pd = new ParsedDocument(new NumericDocValuesField("version", 1), SeqNoFieldMapper.SequenceIDFields.emptySeqID(), "id",
+            "test", null, null, source, XContentType.JSON, null);
+        p = new SlowLogParsedDocumentPrinter(index, pd, 10, true, 3);
+        assertThat(p.toString(), containsString("_failed_to_convert_[Unrecognized token 'invalid'"));
```

Comment:
Can you change this assertion to match the whole error string? It might be a bit fragile to future changes in tests, but since we don't randomize anything here the error should stay stable for a bit and this makes it easiert to understand the whole error structure from reading the test.

Reply:
Thanks! I pushed 9153b6d99567d6b1b68160d739b2656ddf537c74 . We cannot put the while error message here because the internal class ```org.elasticsearch.common.bytes.BytesReference$MarkSupportingStreamInputWrapper``` does not implement a ```toString()``` so the results of ```toString()``` are different each time.